SuperDARN/pydarn
# ACF plotting

I created an ACF plotting class that currently only plots ACF/XCF. Amplitude plotting is soon to come :) 

What does the plotter do: 
- [x] plots real and imaginary
- [x] does not plot missing lags
- [x] plots blanked lags as hollow dots (you can change the marker)
- [x] generates a title for the plot
- [x] labels axis
- [x] plots for a give time/scan, gate number, and beam  

How to test: 
```python 
import pydarn
import matplotlib.pyplot as plt 
from datetime import datetime

reader = pydarn.SDarnRead("20140105.1200.03.cly.rawacf")
data = reader.read_rawacf()
pydarn.ACF.plot(data, beam_num=15, gate_num=16,
                start_time=datetime(2014, 1, 5, 13, 30))
plt.show()
``` 
Expected plot: 

![blank4](https://user-images.githubusercontent.com/6520530/82352706-95d63900-99bb-11ea-8ca4-c4bbbc41be8b.png)
